scilpy/utils/spatial_ops.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+
+def normalize_vector(v):
+    """Normalize a 3D vector."""
+    norm = np.linalg.norm(v)
+    return v / norm if norm != 0 else v
+
+
+def find_perpendicular_vectors(v):
```

Comment:
There is no reference to this function. Is this still relevant and should this be kept ?

Reply:
Old code, will delete! Thank you (same for below)